src/renderer/src/lib/modal/InstallFromFile.svelte

```diff
@@ -37,15 +34,20 @@
 			},
 			{
 				onSuccess: () => {
-					$installedApps.refetch();
 					const filePath = files ? files[0].path : '';
+					$installedApps.refetch().then((apps) => {
+						// Only open the app if it's an app with UI
+						if (filePath.endsWith('.webhapp')) {
+							const newApp = apps.data?.find(
+								(extendedAppInfo) => extendedAppInfo.appInfo.installed_app_id === formData.appId
+							);
+							if (newApp) $openApp.mutate(newApp);
```

Comment:
This is the new logic to open the app immediately after install

Reply:
Visually looks fine. Have you tested it?

Reply:
I have tested it, yes.